idiss-csharp/IdissLibTest/UnitTest1.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Text.Json;
+using IdissLib;
+
+namespace IdissLibTest
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
UnitTest1 sounds autogenerated by an IDE and is not very descriptive. Preferably it should be updated (together with the filename).